# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1374817446 14400
#      Fri Jul 26 01:44:06 2013 -0400
# Node ID 641c47e8bcae182043affd7d7868bf604c00f5f1
# Parent  1d544ac393694280bb63a666828ea4d0aaef2da3
doc: expand on FFT size argument to fftfilt

* fftfilt.m: Explain how the FFT size argument is used and adjusted
internally to meet certain constraints.

diff --git a/scripts/signal/fftfilt.m b/scripts/signal/fftfilt.m
--- a/scripts/signal/fftfilt.m
+++ b/scripts/signal/fftfilt.m
@@ -18,17 +18,21 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} fftfilt (@var{b}, @var{x}, @var{n})
 ##
 ## With two arguments, @code{fftfilt} filters @var{x} with the FIR filter
 ## @var{b} using the FFT.
 ##
 ## Given the optional third argument, @var{n}, @code{fftfilt} uses the
-## overlap-add method to filter @var{x} with @var{b} using an N-point FFT.
+## overlap-add method to filter @var{x} with @var{b} using an @var{n}-point
+## FFT.  The FFT size must be an even power of 2 and must be greater than
+## or equal to the length of @var{b}.  If the specified @var{n} does not
+## meet these criteria, it is automatically adjusted to the nearest value
+## that does.
 ##
 ## If @var{x} is a matrix, filter each column of the matrix.
 ## @seealso{filter, filter2}
 ## @end deftypefn
 
 ## Author: Kurt Hornik <Kurt.Hornik@wu-wien.ac.at>
 ## Created: 3 September 1994
 ## Adapted-By: jwe
